Source/Details/ASDataController.mm

```diff
@@ -133,8 +131,6 @@ - (void)setLayoutDelegate:(id<ASDataControllerLayoutDelegate>)layoutDelegate
 
 - (void)_allocateNodesFromElements:(NSArray<ASCollectionElement *> *)elements
 {
-  ASSERT_ON_EDITING_QUEUE;
```

Comment:
Removing this assertion since it's no longer true under the new treatment group.